wlutil/wlutil.py

```diff
@@ -9,17 +9,48 @@
 import shutil
 import psutil
 import errno
+import pathlib
```

Comment:
When this was first written, pathlib wasn't available (Python 3.4) but now it is so I'm starting a slow switch to using pathlib everywhere. In this file I've just changed the low-hanging fruit that was directly affecting new code. I'll come through in a separate refactoring pass to fully switch over.